src/Tribe/Views/V2/View.php

```diff
@@ -793,8 +805,9 @@ public function prev_url( $canonical = false, array $passthru_vars = [] ) {
 
 		$prev_page  = $this->repository->prev()->order_by( '__none' );
 
-		$paged      = $this->url->get_current_page() - 1;
-		$query_args = $paged > 1
+		$paged           = $this->url->get_current_page() - 1;
+		$query_args      = [];
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: a67d07b7a9a050b0f83261e27a5aac60</sub>